{"name": "#错误的账号、正确的密码、正确的验证码", "status": "passed", "description": "错误的密码", "attachments": [{"name": "log", "source": "bc731c6a-c417-49f5-85ef-d2a7b6df46af-attachment.txt", "type": "text/plain"}, {"name": "stderr", "source": "3ece9de1-57c7-4769-9bf3-f1a643b52eed-attachment.txt", "type": "text/plain"}], "start": 1618312794088, "stop": 1618312798486, "uuid": "74afca86-c8fa-47fd-b65a-83d7ea3a4552", "historyId": "b500888f72c458618cf7391c7f0f990c", "testCaseId": "6931d27b1841f0f05bdca30906246b73", "fullName": "script.test_login.Test_login#test_error_password", "labels": [{"name": "feature", "value": "登录功能"}, {"name": "story", "value": "用例3：错误的账号、正确的密码、正确的验证码"}, {"name": "parentSuite", "value": "script"}, {"name": "suite", "value": "test_login"}, {"name": "subSuite", "value": "Test_login"}, {"name": "host", "value": "DESKTOP-BN9QCCJ"}, {"name": "thread", "value": "14404-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "script.test_login"}], "links": [{"type": "test_case", "url": "https://mail.qq.com/", "name": "qq邮箱"}]}